README.md

```diff
@@ -1,39 +1,74 @@
-# harbor-cli
-LFX mentorship program project original author https://github.com/akshatdalton 
 
-https://github.com/cncf/mentoring/tree/main/programs/lfx-mentorship/2023/01-Mar-May#an-official-golang-api-client-and-cli-for-harbor
-https://mentorship.lfx.linuxfoundation.org/project/7e8cb88a-5b37-471c-8db8-e11907b5a661
+![harbor-3](https://github.com/goharbor/harbor-cli/assets/70086051/835ab686-1cce-4ac7-bc57-05a35c2b73cc)
 
-#Under Contstruction! 
-# How to Install:
+**Welcome to the Harbor CLI project! This powerful command-line tool facilitates seamless interaction with the Harbor container registry. It simplifies various tasks such as creating, updating, and managing projects, registries, and other resources in Harbor.**
 
-- Run `go mod download` to install the dependencies
-- Run `go build -o harbor` to generate the build file
+# Project Scope 🧪
 
-# How to Use:
+The Harbor CLI is designed to enhance your interaction with the Harbor container registry. Built on Golang, it offers a user-friendly interface to perform various tasks related to projects, registries, and more. Whether you're creating, updating, or managing resources, the Harbor CLI streamlines your workflow efficiently.
 
-If you have completed the above steps, now you are all set to use this project.
-<br>
-<br>
-`./harbor --help` or do `./harbor help`:
-```shell
-Official Harbor CLI
+# **Project Features** 🤯
 
-Usage:
-  harbor [command]
+ 🔹 Get details about projects, registries, repositories and more <br>
+ 🔹 Create new projects, registries, and other resources <br>
+ 🔹 Delete projects, registries, and other resources <br>
+ 🔹 Run commands with various flags for enhanced functionality <br>
+ 🔹 More features coming soon... 🚧
 
-Available Commands:
-  completion  Generate the autocompletion script for the specified shell
-  create      create project, registry, etc.
-  delete      delete project, registry, etc.
-  get         get project, registry, etc.
-  help        Help about any command
-  list        list project, registry, etc.
-  login       Log in to Harbor registry
-  update      update registry, etc.
+# Demo Screenshot
 
-Flags:
-  -h, --help   help for harbor
+![demo-1](https://github.com/goharbor/harbor-cli/assets/70086051/63b5f6b7-319b-4c05-968a-59489f7fdd35)
```

Comment:
We don't need a screenshot; we can use Markdown to display available commands. Please check out the latest changes, which include users and repository commands as well.